### DISCUSSION TITLE
Consent to cookies

### DISCUSSION THREAD

**Discussion by michaelcattell** [dwp/design-system#516] [Q&A]
Hello, we'd like to use this open discussion on the use of the **consent to cookies**:

**Some discussion points to get things going:**
Have you used or know where this component has been used?
Do you know if there's been any research or testing done on it or around it?
Do you agree with the user need?
Things we don’t know enough?

**Comment by jwContentDesign** [comment]
The [DWP cookies pattern](https://design-system.dwp.gov.uk/consent-to-cookies) has been replaced by the GOV.UK one, but the [GOV.UK design system cookie page pattern](https://design-system.service.gov.uk/patterns/cookies-page/) does not have the set content that we had on the DWP version. Carer‘s Allowance has an example of where we used the DWP pattern content:
[Carer‘s Allowance cookie policy](https://www.carersallowance.service.gov.uk/ui-citizen-claim/cookie-policy?backto=%2Fui-citizen-claim%2Fallowance-benefits)
[Carer‘s Allowance cookie details](https://www.carersallowance.service.gov.uk/ui-citizen-claim/cookie-details)

The GOV.UK Design System pattern says to work with your organisation’s privacy expert to agree the content.

So, do we still need a DWP pattern to show the standard content we should use for DWP? The GOV.UK pattern does not an example of the full page content, it just has an image example showing part of a page.

**Reply by jonhurrell** [reply]
Thanks for the feedback Jane. We’ll discuss this on Monday and let you know the outcome.

**Reply by jonhurrell** [reply]
Here's a better example of a GDS cookies page (although it may be light), which would tie into the cookies banner component. https://www.notifications.service.gov.uk/cookies. cc @htmlandbacon 
We'd recommend starting with a page like this and see if the content requires the splitting out of details/policy?

**Comment by paulwelsh1** [comment]
Would it be possible to add some guidance for cookie names that include unique strings as part of the name? GA cookies are the main culprit. Looking through a few services, most seem to just ignore that part of the name which I think is sensible, but have seen some others have things like _gat_* 
Noticed a couple of people asking on slack how to approach it too.

**Reply by martinwake** [reply]
We've now updated the guidance to include standard names and descriptions for GA4 cookies: thanks for the suggestion @paulwelsh1 .
